Source/Core/Common/SettingsHandler.cpp

```diff
@@ -11,9 +11,10 @@
 #include <string>
 
 #ifdef _WIN32
+#include <windows.h> //NOLINT
+// The following Windows headers must be included AFTER windows.h.
```

Comment:
Because [the precompiled header code depends on using ln to symlink stuff](https://github.com/dolphin-emu/dolphin/blob/c12f04eefa5262c00751368e40393481a4f90e62/Source/CMakeLists.txt#L26) which obviously doesn't work on Windows.